bin/aggregate-translations.js

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env node
-
-require('../scripts/aggregate-translations/aggregate-translations-cli');
```

Comment:
Can we keep this bin`bin/aggregate-translations.js` file this and point it to require `terra-aggregate-translations/script/aggregate-translations-cli` instead to keep this passive?

Reply:
removing this bin file is passive because [terra-aggregate-transitions](https://github.com/cerner/terra-aggregate-translations/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R31) provides the same bin script which is pull to the `node_modules/.bin/` directory on install